collect_app/src/main/res/values/strings.xml

```diff
@@ -314,32 +314,51 @@
     <!-- Geo string added in feature-geofeatures branch-->
 
     <string name="maps">Maps</string>
-    <string name="map_sdk_selector_title">Mapping SDK</string>
-    <string name="map_sdk_behavior">Mapping SDK</string>
-    <string name="google_maps_sdk">Google Maps SDK</string>
-    <string name="openstreetmap_sdk">OpenStreetMap SDK</string>
-    <string name="mapbox_maps_sdk">Mapbox Maps SDK</string>
-    <string name="show_map_sdk">Show Map SDK Options</string>
-    <string name="openmap_streets">OSM Streets</string>
     <string name="openmap_usgs_topo">USGS National Map Topo</string>
     <string name="openmap_usgs_sat">USGS National Map Hybrid</string>
     <string name="openmap_usgs_img">USGS National Map Imagery</string>
     <string name="openmap_stamen_terrain">Stamen Terrain</string>
     <string name="openmap_cartodb_positron">CartoDB Positron</string>
     <string name="openmap_cartodb_darkmatter">CartoDB Dark Matter</string>
-    <string name="streets">Streets</string>
-    <string name="terrain">Terrain</string>
-    <string name="hybrid">Hybrid</string>
-    <string name="satellite">Satellite</string>
-    <string name="map_basemap_selector_title">Basemaps</string>
-    <string name="show_map_basemap">Show Basemap Options</string>
 
-    <string name="mapbox_streets">Mapbox Streets</string>
-    <string name="mapbox_light">Mapbox Light</string>
-    <string name="mapbox_dark">Mapbox Dark</string>
-    <string name="mapbox_satellite">Mapbox Satellite</string>
-    <string name="mapbox_satellite_streets">Mapbox Satellite Streets</string>
-    <string name="mapbox_outdoors">Mapbox Outdoors</string>
+    <string name="base_layer">Base layer</string>
+    <string name="base_layer_source">Source</string>
+    <string name="base_layer_source_google">Google</string>
```

Comment:
Quick thing for you to consider, @zestyping.

Given that these source names are all organization names, I'm not sure they should be translated. I think they should go in `untranslated.xml` so that they're guaranteed consistent across languages. I imagine some languages might have translations for common brand like Google but I'm sure they also understand the official (English) name.

Then could the map style text be represented as a single parameterized resource? That is, in English it would be `%s map style` and in French it might be `Style de carte %s`.

Reply:
Oh, interesting.  Yeah, that's a fine point.  I'll move them there, and make the parameterized string as you suggest.

Reply:
This is done now, in the commit with the description "Avoid translating basemap source labels...".